### PR TITLE
Add dust check when retiring

### PR DESCRIPTION
**Pull request by kargakis**

### CODE REVIEW

File: contracts/OffsetHelper.sol [hunk 1]
```diff
@@ -287,13 +287,15 @@ contract OffsetHelper is OffsetHelperStorage {
         while (remainingAmount > 0 && i < scoredTCO2Len) {
             address tco2 = scoredTCO2s[i];
             uint256 balance = IERC20(tco2).balanceOf(address(this));
-            i += 1;
-            if (balance == 0) continue;
+            if (balance < 1e15) continue;
```
Comment (lazaralex98) [inline contracts/OffsetHelper.sol:290]: What is this check doing? What is "1e15"?

File: contracts/OffsetHelper.sol [hunk 2]
```diff
@@ -287,13 +287,15 @@ contract OffsetHelper is OffsetHelperStorage {
...
             uint256 amountToRetire = remainingAmount > balance
                 ? balance
                 : remainingAmount;
             IToucanCarbonOffsets(tco2).retire(amountToRetire);
             remainingAmount -= amountToRetire;
+            unchecked {
```
[the remark this answers is not in the document]
Comment (kargakis) [reply contracts/OffsetHelper.sol:296]: correct

File: contracts/OffsetHelper.sol (hunk 1 above)
Comment (kargakis) [reply contracts/OffsetHelper.sol:290]: This checks whether `balance` is less than `1e15` which is a concise way to write 1000000000000000, or 0.001 TCO2 or 1 kilo of co2.

https://stattrek.com/statistics/dictionary.aspx?definition=E-Notation

Comment (lazaralex98) [reply contracts/OffsetHelper.sol:290]: Oh, it's less than a kilo amounts. Got it.

**Pull request merged**